code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/wrappedform/WrappedFormFieldFormModel.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2022 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/org/documents/edl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+import {Button, GroupBox, SmartField, WrappedFormField} from '@eclipse-scout/core';
+import {WrappedFormLookupCall} from '../index';
+
+export default () => ({
```

Comment:
Please use the same pattern as for every other widget: The widget itself is in the top box, on the bottom is the configuration box with the properties, Actions and Events for the widget (WrappedFormField). InnerFormField is a property -> it belongs into the properties box